### ISSUE TITLE
fix `SS_plots` when Dirichlet-multinomial is on

### ISSUE THREAD

**Issue by James-Thorson** [r4ss/r4ss#186]
I'm exploring a case-study involving the Dirichlet-multinomial for a single fleet for length-comps.  `SS_output` worked well, but `SS_plots` threw an error with message:

```R
Starting fit to age comp plots (group 17)
[1] "Francis Weights - age: 6_N_TRI_Late: 1.8623 (0.9772-Inf)"
Starting fit to conditional age-at-length comp plots (group 18)
Error in plot.window(...) : need finite 'ylim' values
In addition: Warning messages:
1: In min(x) : no non-missing arguments to min; returning Inf
2: In max(x) : no non-missing arguments to max; returning -Inf
```
Any thoughts @taylori?  Its not a big deal for my currently purposes, but presumably will need some future love.

**Comment by taylori** [comment]
Thanks for reporting this. The minimum standard is for nothing to crash. If you could share with me your model input files (and Report.sso and CompReport.sso to save me running the model again), I'll try to sort this out quickly.
I'm assuming that it would still be useful to see the Francis tuning plots showing mean age or length even for models that use the D-M likelihood, but it's worth confirming that the uncertainty intervals around those means are accurate and meaningful when the D-M likelihood is used. But that's a lower priority than getting the function to not crash.

**Comment by taylori** [comment]
Fix in commit 72604dcc11ec58a4437798fd5ce76fd8a47167d9 allows function to play through. The issue was caused by a variance adjustments factor of 0 applied to input sample sizes for conditional age data for one of the fleets causing the sample sizes to be 0 for all years. The function should no longer crash, but it's probably better practice to set adjustment to a very small positive number rather than 0, or set the likelihood weight (lambda) to 0 instead.
